subprojects/ide-native/src/integTest/groovy/org/gradle/ide/visualstudio/VisualStudioSoftwareModelSingleProjectIntegrationTest.groovy

```diff
@@ -30,13 +30,17 @@ import org.gradle.util.TestPrecondition
 import spock.lang.Issue
 
 import static org.gradle.nativeplatform.fixtures.ToolChainRequirement.VISUALCPP
+import static org.gradle.nativeplatform.fixtures.ToolChainRequirement.WINDOWS_GCC
+import static org.junit.Assume.assumeFalse
 
 class VisualStudioSoftwareModelSingleProjectIntegrationTest extends AbstractVisualStudioIntegrationSpec {
     private final Set<String> projectConfigurations = ['win32Debug', 'win32Release', 'x64Debug', 'x64Release'] as Set
 
     def app = new CppHelloWorldApp()
 
     def setup() {
+        assumeFalse(toolChain.meets(WINDOWS_GCC))
```

Comment:
What stops us from running these tests with Cygwin?

Reply:
Our CI servers don't have cygwin/mingw installs that support 64-bit compilation.  I believe we have an issue out there to install them, but they aren't available at the moment.  I'd rather just disable this test for those toolchains than change the variants (although that would be another option).  There just just doesn't seem to be that much value to running these tests with those toolchains.